conference.js

```diff
@@ -1891,6 +1897,7 @@ export default {
             }
 
             APP.UI.addListener(UIEvents.SELECTED_ENDPOINT, id => {
+                APP.API.notifyOnStageParticipantChanged(id);
```

Comment:
How does the always on top video know in other situations that the participant on stage has changed?

Reply:
Do you mean shared video, etherpad, etc? It doesn't know I suppose.

About shared video, etherpad, etc use cases I and @yanas agreed to handle them in future with another PR.

Reply:
How does it handle switching video right now when the active speaker changes? I'd like to review that flow too.

Reply:
updateLargeVideo() -> UIEvents.SELECTED_ENDPOINT -> APP.API.notifyOnStageParticipantChanged.

This handles any updates of the large video.

Or am I missing something?

Reply:
I should be more clear. I was asking how has it worked this whole time before the changes you're proposing in this pull request. I figured it out now by reviewing your electron changes and looking at current master. Electron was reaching into the iframe and doing a dom query for the video element and listening for play events. With these changes you moved the querying into jitsi-meet so that electron is notified of the change and calls back into the api to get the video element.

Reply:
Yeah! Sorry that I miss understood you and didn't explained!